docs/rules/no-nonoctal-decimal-escape.md

```diff
@@ -0,0 +1,58 @@
+# Disallow `\8` and `\9` escape sequences in string literals (no-nonoctal-decimal-escape)
+
+Although not being specified in the language until ECMAScript 2021, `\8` and `\9` escape sequences in string literals were allowed in most JavaScript engines, and treated as "useless" escapes:
```

Comment:
Can you add a link to the appropriate location in the spec both in this paragraph and under a "Further Reading" heading at the bottom?

Reply:
Makes sense, done!